lemur/endpoints/service.py

```diff
@@ -51,6 +51,20 @@ def get_by_name(name):
     return database.get(Endpoint, name, field="name")
 
 
+def get_by_name_and_source(name, source):
+    """
+    Retrieves and endpoint by it's name and source.
```

Comment:
minor language: 
`Retrieves an endpoint by it's name and source`

Reply:
Fixed :)